README.md

```diff
@@ -49,16 +49,16 @@ Fill this in for each provider
 
 ## Developing the Provider
 
-If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](#requirements) above).
+Run these commands in order to test a single test:
 
-To compile the provider, run `go install`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
-
-To generate or update documentation, run `go generate`.
-
-In order to run the full suite of Acceptance tests, run `make testacc`.
+```bash
+make install
+find . -type f -name "*.terraform.lock.hcl" -exec rm -f {} +
+go test -v ./... -run "TestCanChangeIsLive"
```

Comment:
Just the one test?

Reply:
Yeah this runs a single test, then the next block runs all tests

Reply:
Whoops, sorry didn't read the whole block